src/mx_bluesky/hyperion/experiment_plans/load_centre_collect_full_plan.py

```diff
@@ -96,14 +100,13 @@ def plan_with_callback_subs():
 
         multi_rotation.rotation_scans.clear()
 
+        sweep_state: RotationDirection | None = None
```

Comment:
Could: Couldn't you just set this to
```suggestion
        sweep_state: RotationDirection = rotation_template[0].rotation_direction
```
Feels like it makes things a bit more readable than having to have `None` as a special value that that you deal with later.